### PR TITLE
Add `purge(Project|Repository)` API

### PR DESCRIPTION
**Pull request by ikhoon**
Motivation:
New `purgeProject` and `purgeRepository` API were added in Central Dogma [0.41.0](https://github.com/line/centraldogma/releases/tag/centraldogma-0.41.0) release. To keep up with it, we need to apply the new API.

Modification:
* Add `purgeProject` and `purgeRepository` and their test codes.
* Reformat codes.

Result:
Users are able to use purge API in go client.

### CODE REVIEW

File: repository_service.go [hunk 1]
```diff
@@ -24,11 +24,11 @@ type repositoryService service
 
 // Repository represents a repository in the Central Dogma server.
 type Repository struct {
-	Name         string  `json:"name"`
-	Creator      Author  `json:"creator,omitempty"`
-	HeadRevision int     `json:"headRevision,omitempty"`
-	URL          string  `json:"url,omitempty"`
-	CreatedAt    string  `json:"createdAt,omitempty"`
```
Comment (minwoox) [inline repository_service.go]: Very good!

File: watch_service_test.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ func TestWatchFile(t *testing.T) {
 			time.Sleep(100 * time.Millisecond)
 			if notModifiedResponse {
 				w.WriteHeader(http.StatusNotModified)
-				notModifiedResponse = false;
```
Comment (minwoox) [inline watch_service_test.go]: Nice catch!

Comment (ikhoon) [reply watch_service_test.go]: It was fixed by `go fmt`, not me. 😜

Comment (minwoox) [reply watch_service_test.go]: Then, nice catch, `go fmt`!

File: project_service.go [hunk 3]
```diff
@@ -66,6 +66,21 @@ func (p *projectService) remove(ctx context.Context, name string) (int, error) {
 	return httpStatusCode, nil
 }
 
+func (p *projectService) purge(ctx context.Context, name string) (int, error) {
+	u := defaultPathPrefix + "projects/" + name + "/removed"
```
Comment (linxGnu) [inline project_service.go]: Use `u := fmt.Sprintf("` instead

**Comment by minwoox** [comment]
> @minwoox I think we could make url in better form with path.Join later.
So may be next PR. See also: golang.org/pkg/path

Didn't know that there's a package for this. Looking forward to it. Thanks!

File: project_service.go (hunk 3 above)
Comment (ikhoon) [reply project_service.go]: @linxGnu Thanks, I will fix all similar cases in this file. :-)

**Comment by trustin** [comment]
Thanks, @ikhoon!

**Pull request merged**